package.json

```diff
@@ -1,101 +1,99 @@
-    },
-    "devDependencies": {
-        "@angular-devkit/build-angular": "~0.7.1",
-        "@angular-devkit/build-ng-packagr": "~0.7.1",
-        "@angular-devkit/schematics": "^0.7.1",
-        "@angular/cli": "~6.1.1",
-        "@angular/compiler-cli": "^6.1.0",
-        "@angular/language-service": "^6.1.0",
-        "@angularclass/hmr": "^2.1.3",
-        "@types/jasmine": "~2.8.8",
-        "@types/jasminewd2": "~2.0.3",
-        "@types/node": "~10.5.4",
-        "@types/webpack-env": "^1.13.6",
-        "browser-sync": "^2.24.6",
-        "codelyzer": "~4.4.2",
-        "coveralls": "^3.0.2",
-        "gulp": "^3.9.1",
-        "gulp-concat": "^2.6.1",
-        "gulp-postcss": "^7.0.1",
-        "gulp-sass": "^4.0.1",
-        "gulp-shell": "^0.6.5",
-        "gulp-sourcemaps": "^2.6.4",
-        "gulp-uglify": "^3.0.1",
-        "hammer-simulator": "git+https://github.com/hammerjs/simulator.git",
-        "jasmine": "^3.1.0",
-        "jasmine-core": "~3.1.0",
-        "jasmine-spec-reporter": "~4.2.1",
-        "karma": "~2.0.5",
-        "karma-chrome-launcher": "~2.2.0",
-        "karma-coverage-istanbul-reporter": "~2.0.1",
-        "karma-jasmine": "~1.1.2",
-        "karma-jasmine-html-reporter": "~1.2.0",
-        "karma-junit-reporter": "^1.2.0",
-        "karma-spec-reporter": "~0.0.32",
-        "lunr": "^2.3.3",
-        "ng-packagr": "^3.0.6",
-        "protractor": "~5.3.0",
-        "sassdoc": "^2.5.0",
-        "stylelint": "^8.4.0",
-        "stylelint-scss": "^2.5.0",
-        "themeleon": "^3.0.2",
-        "ts-node": "~7.0.0",
-        "tsickle": ">=0.25.5",
-        "tslib": "^1.9.3",
-        "tslint": "~5.11.0",
-        "typedoc": "^0.13.0",
-        "typedoc-plugin-localization": "^1.2.1",
-        "typescript": "~2.7.2",
-        "webpack-sources": "1.2.0"
-    }
+  "name": "igniteui-dev-demos",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve --open --configuration hmr",
+    "build": "ng build",
+    "test": "ng test",
+    "lint": "ng lint",
+    "e2e": "ng e2e",
+    "test:lib": "ng test igniteui-angular --watch=false --no-progress --code-coverage",
+    "test:lib:azure": "ng test igniteui-angular --watch=false --no-progress --code-coverage --karma-config=./projects/igniteui-angular/karma.azure.conf.js",
+    "test:lib:watch": "ng test igniteui-angular",
+    "test:migration": "node -r ts-node/register ./node_modules/jasmine/bin/jasmine.js ./projects/igniteui-angular/migrations/**/*.spec.ts",
+    "build:lib": "ng build igniteui-angular --prod && gulp build-style",
+    "build:style": "gulp build-style",
+    "build:migration": "gulp copy-migrations && tsc --listEmittedFiles --project ./projects/igniteui-angular/migrations/tsconfig.json",
+    "build:docs:export": "gulp typedoc-build:export",
+    "build:docs:import": "gulp typedoc-build:import",
+    "build:typedoc:ja:localization": "gulp typedoc-build:doc:ja:localization && gulp clean-translations:localization:repo",
```

Comment:
This was probably due to conflict resolutions, but the `build:typedoc` commands in master are different and there's 4 of them:
```
"build:typedoc:ja:production":
"build:typedoc:ja:staging":
"build:typedoc:en:production":
"build:typedoc:en:staging":
```